.bash_profile.khan

```diff
@@ -40,3 +40,8 @@ if ! which gcloud >/dev/null; then
         source "$GCLOUD_COMPLETION_FILE"
     fi
 fi
+
+# Add a brew86 alias if we're on an ARM architecture Mac
+if [ "$(uname -s)" = "Darwin" ] && [ "$(uname -m)" = "arm64" ]; then
+  alias brew86="arch -x86_64 /usr/local/bin/brew $@"
+fi
```

Comment:
This will work on bash, but not fish (and probably not ksh?)

Why not just create a `bash86.sh` script?

(Maybe I'm overlooking some logic here?)

Reply:
This is in the `.bash_profile.khan` so that seems reasonable, but I do like the idea of making it a script. The only problem is it would be confusing to add the script to linux machines, so would want to be careful about that, but that does seem pretty reasonable!